ReactNativeClient/lib/components/screens/encryption-config.js

```diff
@@ -46,6 +46,16 @@ class EncryptionConfigScreenComponent extends BaseScreenComponent {
 	componentDidMount() {
 		this.isMounted_ = true;
 		shared.componentDidMount(this);
+
+		const masterKeys = this.props.masterKeys;
+
+		for (let i = 0; i < masterKeys.length; i++) {
+			const mk = masterKeys[i];
+
+			this.setState(prevState =>({
+				passwordHide: [{ id: mk.id, [mk.id]: true }, ...prevState.passwordHide],
+			}));
+		}
```

Comment:
We shouldn't show more than one password at once, so is no need to store a list, just the ID of the currently shown master key password.